### DISCUSSION TITLE
Authorization failed... for list_discovered_hosts.py

### DISCUSSION THREAD

**Discussion by afroz1** [CrowdStrike/falconpy#683] [Q&A] [answered]
#python3 list_discovered_hosts.py  -k $FALCON_CLIENT_ID -s $FALCON_CLIENT_SECRET -r  -f simple
[403] access denied, authorization failed

seems it's not able to get the Token by passing ID and SECRET.

**Comment by jshcodes** [comment, marked as answer]
Hi @afroz1 -

Thank you for the question! 

For this sample, you will need to have the **READ** scope for Discover enabled on your API key. In order to access the Discover API, you will need the Discover module enabled within your Falcon tenant. You should be able to confirm this module's status in the Falcon console.

**Reply by afroz1** [reply]
How would i need to define the module's status in the Falcon console.

**Reply by jshcodes** [reply]
If you're provisioned for Discover, you will be able to find links for it in the Console navigation.  (It does not display when it is not licensed.)

You could also check the CrowdStrike store, your provisioning status would be reflected there if you have an active trial.
